### PR TITLE
🔝 Change ethers and similar package versions to the newest ones

### PR DESCRIPTION
**Pull request by mj426382**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
   },
   "dependencies": {
     "@changesets/cli": "^2.14.1",
-    "ethers": "^5.5.4",
+    "ethers": "5.6.0",
```
Comment (dmaretskyi) [inline package.json:19]: ^5.6.2

Comment (mj426382) [inline package.json:19]:
```suggestion
    "ethers": "^5.6.2",
```

File: package.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
...
     "prettier": "2.1.2",
     "wsrun": "^5.2.1"
   },
   "resolutions": {
-    "ethers": "^5.5.4"
+    "ethers": "5.6.0"
```
Comment (mj426382) [inline package.json:24]:
```suggestion
    "ethers": "^5.6.2"
```

File: packages/coingecko/package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@ethersproject/contracts": "5.6.0",
-    "@ethersproject/providers": "5.6.0",
+    "@ethersproject/providers": "5.6.2",
```
Comment (mj426382) [inline packages/coingecko/package.json:35]:
```suggestion
    "@ethersproject/providers": "^5.6.2",
```

File: packages/core/package.json [hunk 4]
```diff
@@ -36,16 +36,16 @@
     "testing.d.ts"
   ],
   "dependencies": {
-    "@ethersproject/abi": "5.6.0",
+    "@ethersproject/abi": "5.6.1",
```
Comment (mj426382) [inline packages/core/package.json:39]:
```suggestion
    "@ethersproject/abi": "^5.6.1",
```

File: packages/core/package.json [hunk 5]
```diff
@@ -36,16 +36,16 @@
...
     "@ethersproject/abstract-provider": "5.6.0",
     "@ethersproject/bignumber": "5.6.0",
     "@ethersproject/constants": "5.6.0",
     "@ethersproject/contracts": "5.6.0",
-    "@ethersproject/providers": "5.6.0",
+    "@ethersproject/providers": "5.6.2",
```
Comment (mj426382) [inline packages/core/package.json:44]:
```suggestion
    "@ethersproject/providers": "^5.6.2",
```

File: packages/core/package.json [hunk 6]
```diff
@@ -36,16 +36,16 @@
...
     "@ethersproject/wallet": "5.6.0",
     "@metamask/detect-provider": "^1.2.0",
     "@uniswap/token-lists": "^1.0.0-beta.27",
-    "ethers": "^5.5.4",
+    "ethers": "5.6.2",
```
Comment (mj426382) [inline packages/core/package.json:48]:
```suggestion
    "ethers": "^5.6.2",
```

File: packages/docs/package.json [hunk 7]
```diff
@@ -24,7 +24,7 @@
     "@docusaurus/preset-classic": "2.0.0-beta.17",
     "@docusaurus/remark-plugin-npm2yarn": "^2.0.0-beta.17",
     "@ethersproject/contracts": "5.6.0",
-    "@ethersproject/providers": "5.6.0",
+    "@ethersproject/providers": "5.6.2",
```
Comment (mj426382) [inline packages/docs/package.json:27]:
```suggestion
    "@ethersproject/providers": "^5.6.2",
```

File: packages/example/package.json [hunk 8]
```diff
@@ -19,7 +19,7 @@
     "@ethersproject/contracts": "5.6.0",
     "@web3-react/walletconnect-connector": "6.2.4",
     "@ethersproject/units": "5.6.0",
-    "@ethersproject/abi": "5.6.0",
+    "@ethersproject/abi": "5.6.1",
```
Comment (mj426382) [inline packages/example/package.json:22]:
```suggestion
    "@ethersproject/abi": "^5.6.1",
```

File: packages/example/package.json [hunk 9]
```diff
@@ -36,7 +36,7 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/react-router-dom": "^5.1.7",
-    "@ethersproject/providers": "5.6.0",
+    "@ethersproject/providers": "5.6.2",
```
Comment (mj426382) [inline packages/example/package.json:39]:
```suggestion
    "@ethersproject/providers": "^5.6.2",
```

File: packages/extension/package.json [hunk 10]
```diff
@@ -15,9 +15,9 @@
     "build-storybook": "build-storybook -s ./static"
   },
   "dependencies": {
-    "@ethersproject/abi": "5.6.0",
+    "@ethersproject/abi": "5.6.1",
```
Comment (mj426382) [inline packages/extension/package.json:18]:
```suggestion
    "@ethersproject/abi": "^5.6.1",
```

File: packages/extension/package.json [hunk 11]
```diff
@@ -15,9 +15,9 @@
...
     "@ethersproject/address": "5.6.0",
-    "ethers": "^5.5.4",
+    "ethers": "5.6.2",
```
Comment (mj426382) [inline packages/extension/package.json:20]:
```suggestion
    "ethers": "^5.6.2",
```

File: packages/testing/package.json [hunk 12]
```diff
@@ -28,12 +28,12 @@
   "dependencies": {
     "@ethersproject/constants": "5.6.0",
     "@ethersproject/contracts": "5.6.0",
-    "@ethersproject/providers": "5.6.0",
+    "@ethersproject/providers": "5.6.2",
```
Comment (mj426382) [inline packages/testing/package.json:31]:
```suggestion
    "@ethersproject/providers": "^5.6.2",
```

File: packages/testing/package.json [hunk 13]
```diff
@@ -28,12 +28,12 @@
...
     "@types/react": "17.0.1",
     "@types/react-dom": "17.0.1",
     "@usedapp/core": "workspace:*",
     "ethereum-waffle": "4.0.0-alpha.0",
-    "ethers": "^5.5.4",
+    "ethers": "5.6.2",
```
Comment (mj426382) [inline packages/testing/package.json:36]:
```suggestion
    "ethers": "^5.6.2",
```

**Review by rzadp — APPROVED** [review]

**Review by dmaretskyi — APPROVED** [review]

**Pull request merged**